tululbot/commands.py

```diff
@@ -0,0 +1,95 @@
+from urllib.parse import urlencode
```

Comment:
The fact that a command can return a tuple whose second element is whether or not to display preview should be documented somewhere... maybe in this module?